hbase_storage/hbase1.2.0/src/main/java/com/splicemachine/storage/Hbase10PartitionCache.java

```diff
@@ -40,6 +40,7 @@ public Hbase10PartitionCache(){ }
     public void invalidate(TableName tableName) throws IOException{
         partitionCache.invalidate(tableName);
         ((HConnection)HBaseConnectionFactory.getInstance(config).getConnection()).clearRegionCache(tableName);
+        ((HConnection)HBaseConnectionFactory.getInstance(config).getNoRetryConnection()).clearRegionCache(tableName);
```

Comment:
This has to be replicated in the Hbase10PartitionCache.java for the other platforms